influxql/token.go

```diff
@@ -7,28 +7,29 @@ import (
 // Token is a lexical token of the InfluxQL language.
 type Token int
 
+// These are a comprehensive list of InfluxQL language tokens.
 const (
-	// Special tokens
+	// ILLEGAL Token, EOF, WS are Special IQL tokens.
 	ILLEGAL Token = iota
 	EOF
 	WS
 
-	literal_beg
-	// Literals
-	IDENT        // main
-	NUMBER       // 12345.67
-	DURATION_VAL // 13h
-	STRING       // "abc"
-	BADSTRING    // "abc
-	BADESCAPE    // \q
-	TRUE         // true
-	FALSE        // false
-	REGEX        // Regular expressions
-	BADREGEX     // `.*
-	literal_end
+	literalBeg
+	// IDENT and the following are IQL Literal tokens.
+	IDENT       // main
+	NUMBER      // 12345.67
+	DURATIONVAL // 13h
```

Comment:
This is a breaking change to the external API. I doubt many people (if any) are using this constant but it's worth noting. @jwilder What do you think?